src/penn_chime/parameters.py

```diff
@@ -11,7 +11,7 @@
 
 # (0.02, 7) is 2%, 7 days
 # be sure to multiply by 100 when using as a default to the percent widgets!
-RateLos = namedtuple("RateLos", ("rate", "length_of_stay"))
+RateDays = namedtuple("RateDays", ("rate", "days"))
```

Comment:
```suggestion
DaysAtRate= namedtuple("PeriodAtRate", ("rate", "days"))
```
Not necessary for this PR but I find the name confusing.

Reply:
This is actual a tuple of (rate of disposition within the infected) and (average days of disposition), so the rate is something else.